api/libs/scenarios-planning-unit/src/scenarios-pu-pa-data.geo.entity.ts

```diff
@@ -0,0 +1,43 @@
+import {
+  Column,
+  Entity,
+  JoinColumn,
+  ManyToOne,
+  OneToOne,
+  PrimaryGeneratedColumn,
+  RelationId,
+} from 'typeorm';
+import { ScenariosPlanningUnitGeoEntity, ScenariosPuCostDataGeo } from '@marxan/scenarios-planning-unit';
+import { PlanningUnitsGeom } from '@marxan-jobs/planning-unit-geometry/planning-units.geo.entity';
+
+@Entity({
+  name: 'scenarios_pu_data',
+})
+export class ScenariosPuPaDataGeo extends ScenariosPlanningUnitGeoEntity {
+
+  @Column({
+    type: 'float8',
+    nullable: true,
+    name: 'protected_area',
+  })
+  protectedArea!: number;
+
+  @OneToOne(() => PlanningUnitsGeom, PlanningUnitsGeom => PlanningUnitsGeom.id, {
+    onDelete: 'CASCADE',
+  })
+  @JoinColumn({
+    name: 'pu_geom_id',
+    referencedColumnName: 'id',
+  })
+  planning?: PlanningUnitsGeom | null;
+
+  @OneToOne(() => ScenariosPuCostDataGeo, ScenariosPuCostDataGeo => ScenariosPuCostDataGeo.id, {
+    onDelete: 'CASCADE',
+  })
+  @JoinColumn({
+    name: 'pu_geom_id',
+    referencedColumnName: 'id',
+  })
+  cost?: PlanningUnitsGeom | null;
```

Comment:
same as above - this is not really a `cost`, but something like `costData` 😬 , and same as above for the cascade (I think).

Not sure if a view could maybe also help with all these joins, by giving us a denormalized but immediately-usable view entity (not for this PR unless you think this is possible, beneficial *and* easy to get right 😅 )

Reply:
i was avoiding a view at this stage as the idea is to plugin this joins as needed and requested from the FE